.github/workflows/check.yml

```diff
@@ -34,6 +34,7 @@ jobs:
       - uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af
         with:
           toolchain: stable
+          target: ${{ matrix.target }}
```

Comment:
It seems that we need the musl-tools. We should add a step to install it. 

`sudo apt-get install -y musl-tools`

Reply:
That did it, thanks!